lib/evss/claims_service.rb

```diff
@@ -22,10 +24,14 @@ def submit_5103_waiver(claim_id)
       }.to_json
     end
 
+    def self.breakers_service
+      BaseService.create_breakers_service(name: 'EVSS/Claims', url: BASE_URL)
+    end
+
     protected
 
     def base_url
-      "#{ENV['EVSS_BASE_URL']}/wss-claims-services-web-3.0/rest"
+      BASE_URL
```

Comment:
`base_url` is a method so that the sub-classes can override it. If it's going to have the same content in all sub-classes, we can move it to the base class.